docs/components/plugins/APISection.tsx

```diff
@@ -31,10 +32,15 @@ const filterDataByKind = (
 const renderAPI = (
   packageName: string,
   version: string = 'unversioned',
-  apiName?: string
+  apiName?: string,
+  isTestMode: boolean = false
 ): JSX.Element => {
   try {
-    const data = require(`~/public/static/data/${version}/${packageName}.json`).children;
+    // Note: When the path prefix is interpolated the Next fails to locate the file,
```

Comment:
cc @byCedric in case you have any idea

Reply:
This is the error which pops out:
```
instrument.js?ea14:109 Error: Cannot find module '~/public/static/data/unversioned/expo-apple-authentication.json' 
at webpackEmptyContext (eval at ./components/plugins sync recursive ^~.*$ (apple-authentication.js?ts=1626972324629:634), <anonymous>:2:10)
```
So it also might be a Webpack issue.

Reply:
Dynamic imports aren't handled very well by Webpack. It's because Webpack doesn't know the possible values of the variables used in the paths. For packages, Webpack solves this by just adding all files within that package. I think this might be harder to solve.

It could also be related to the `~/` path not being resolved properly in the tests by Jest.

Reply:
> It could also be related to the ~/ path not being resolved properly in the tests by Jest.

That was my first impression about this issue, but error based on Webpack part reduced my confidence in my knowledge what is actually happening in there. 🙂